components/gridlayout/items.md

```diff
@@ -135,23 +135,26 @@ The `Column` parameter controls in which column the `GridLayoutItem` will reside
 
 ## ColumnSpan
 
-The `ColumnSpan` parameter defines the how many columns the item will occupy. 
+The `ColumnSpan` parameter defines the how many columns the item will occupy.
+
+When you specify the desired `ColumnSpan`, you should also provide the `Column` that will serve as a start position for the item taking several columns.
```

Comment:
"provide the `Column`" sounds ambiguous and I had to read it several times and guess what it means.

```suggestion
When you set the desired `ColumnSpan`, you should also set the `Column` parameter to specify the start position for the spanned item.
```